src/ocgis/ops/parms/definition.py

```diff
@@ -72,6 +72,7 @@ class Aggregate(base.BooleanParameter):
     meta_false = 'Selected geometries are not aggregated (unioned).'
 
 
+# This is obsolete following new netCDF driver with support for discrete geoms.
```

Comment:
We're going to want to leave this in there in case of future logic associated with this argument. It's also used for operations metadata.

Reply:
The allow edit from maintainers was checked. You should have access but I've added you to the collaborators for my fork.

Reply:
Comment removed.